src/components/ErrorBoundary.tsx

```diff
@@ -44,15 +45,21 @@ export const ErrorBoundary = ({
     if (!sentryDsn && !sentryInit) {
       return;
     }
+    if (initCalled.current) {
+      throw 'Sentry.init was already called';
+    }
 
     Sentry.init(sentryInit || {
       dsn: sentryDsn,
       environment: window.location.hostname,
       integrations: [
-        new Sentry.BrowserTracing(),
+        Sentry.browserTracingIntegration(),
+        Sentry.extraErrorDataIntegration()
       ],
+      tracesSampleRate: 0.1,
     });
-  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+    initCalled.current = true;
```

Comment:
not that it matters, because there is no async here, but i would put this immediately after the check above. otherwise if there was async stuff in here you could get two concurrent calls going.